src/status_im/utils/ethereum/resolver.cljs

```diff
@@ -0,0 +1,16 @@
+(def default-address "0x0000000000000000000000000000000000000000")
+(def default-hash "0x0000000000000000000000000000000000000000000000000000000000000000")
+
```

Comment:
Why this? Can't we do the same way as it's done for others field in the `ens` namespace?

Reply:
oh right if we're going with `content` field we can use same way